README.md

```diff
@@ -0,0 +1,13 @@
+# How to update
+
+  - The repo of this online tutorial content is at https://github.com/mozilla-b2g/casting-tutorial.
+    Please send pull request to this specific repo.
+
+  - Accroding to the bug 1215006: https://bugzilla.mozilla.org/show_bug.cgi?id=1215006,
+    we serve the turotial content at https://casttv.services.mozilla.com/.
+    Please file a bug to jthomas@mozilla.com to update the website.
+
+# The Fling tutorial
```

Comment:
Will change to a generic description to avoid confusion and leave a right link to let maintainer find the corresponding TV app.